src/model/CplacePlugin.ts

```diff
@@ -52,14 +52,14 @@ export default class CplacePlugin {
 
         this.repo = path.basename(path.dirname(path.resolve(pluginDir)));
         this.assetsDir = path.resolve(pluginDir, 'assets');
-        this.hasTypeScriptAssets = fs.existsSync(path.resolve(this.assetsDir, 'ts'));
+        this.hasTypeScriptAssets = fs.existsSync(path.resolve(this.assetsDir, 'ts', 'app.ts'));
         this.hasTypeScriptE2EAssets = false;
         const e2ePath: string = path.resolve(this.assetsDir, 'e2e');
         if (fs.existsSync(e2ePath)) {
             this.hasTypeScriptE2EAssets = glob.sync(`${e2ePath}/**/*.ts`).length > 0;
         }
 
-        this.hasLessAssets = fs.existsSync(path.resolve(this.assetsDir, 'less'));
+        this.hasLessAssets = (glob.sync(`${this.assetsDir}/*.less`).length > 0 || glob.sync(`${this.assetsDir}/**/*.less`).length > 0);
```

Comment:
The first glob is included in the second glob so it suffices to just use the `**/*.less` variant. Please also join the paths with `path.join(this.assetsDir, '**', '*.less')` to circumvent having maybe multiple slashes somewhere or slashes instead of backslashes etc.